packages/itwinui-react/src/core/Table/TablePaginator.tsx

```diff
@@ -326,18 +330,19 @@ export const TablePaginator = (props: TablePaginatorProps) => {
               if (visibleCount === 1) {
                 return pageButton(focusedIndex);
               }
+
               return (
                 <>
                   {startPage !== 0 && (
                     <>
                       {pageButton(0, 0)}
-                      {ellipsis}
+                      {showStartEllipsis && ellipsis}
```

Comment:
```suggestion
                      {showStartEllipsis ? ellipsis : null}
```

Reply:
Yes, we want to avoid rendering `undefined` or `false`, even if it works in the newer React versions. Using the ternary expression is "more correct". See also: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

We can update all instances in a separate PR, and maybe update our style guide. A lint rule would also be nice to help avoid these issues before committing.